pkg/prowloader/testconversion/testconversion.go

```diff
@@ -0,0 +1,85 @@
+package testconversion
+
+import (
+	"github.com/openshift/sippy/pkg/apis/junit"
+	"github.com/openshift/sippy/pkg/apis/prow"
+	v1 "github.com/openshift/sippy/pkg/apis/sippyprocessing/v1"
+	testgridv1 "github.com/openshift/sippy/pkg/apis/testgrid/v1"
+	"github.com/openshift/sippy/pkg/db/models"
+	"github.com/openshift/sippy/pkg/synthetictests"
+	"github.com/openshift/sippy/pkg/testidentification"
+)
+
+func ConvertProwJobRunToSyntheticTests(pj prow.ProwJob, tests map[string]*models.ProwJobRunTest, manager synthetictests.SyntheticTestManager) (*junit.TestSuite, v1.JobOverallResult) {
+	jrr := v1.RawJobRunResult{
+		Job:       pj.Spec.Job,
+		Failed:    pj.Status.State != prow.SuccessState,
```

Comment:
Even if filtered elsewhere this looks a little risky, Pending and Triggered could both trigger as "failed" here. I'd go more explicit. Is aborted a failure as well? (probably yes for our purposes)

Reply:
Fair enough, fixed in last commit.